src/main/java/game/domain/Board.java

```diff
@@ -0,0 +1,48 @@
+package game.domain;
+
+import java.util.Arrays;
+
+public class Board {
+    private static final String INVALID_INPUT = "입력이 잘못되었습니다, 명령(h, j, k, l, q)";
+    private final Positions positions;
+    private final OrderChecker orderChecker;
+    private Pointer pointer;
+    private Moves moves;
+
+    public Board(Numbers numbers) {
+        this.positions = new Positions(numbers);
+        this.orderChecker = new OrderChecker();
+        this.pointer = Pointer.from(positions);
+        this.moves = new Moves();
+    }
+
+    /**
+     * 입력한 방향으로 포인터를 이동한다.
+     * @param input 입력방향
+     */
+    public void movePointer(String input) {
+        Directions directions = Directions.from(input);
+        if (directions == null) {
+            System.out.println(INVALID_INPUT);
+            return;
+        }
+        int newRow = directions.moveRow(pointer);
+        int newCol = directions.moveCol(pointer);
+
+        positions.swapPositions(pointer, newRow, newCol);
```

Comment:
```suggestion
        positions.swapPositions(pointer, swapPointer);
```

newRow, newCol은 newPointer가 만들어지는 것 같은데, 차라리 swapPosition에 기존 pointer와 newPointer를 넣어서 교체해주는게 좋지 않을까합니다 :) 

포인터끼리 변경하는 것인데, newRow, newCol를 넣는게 자연스럽지는 않네요.

Reply:
말씀하신대로 swap이면 동일한 개념이 변경되어야하는데 일치하지 않았군요. 
[9445110](https://github.com/etff/fifteen-puzzle/pull/1/commits/9445110c99dfdd4114b0ddd8533b51edf12e1792) 에 수정했습니다.